### PR TITLE
remove hidden attr for checkboxes and radio's

### PR DESCRIPTION
**Pull request by moh-moola**

### CODE REVIEW

**Comment by moh-moola** [comment]
screenshot_2018-10-17_10-08-38

File: tuneme/styles/tuneme/_base.scss [hunk 1]
```diff
@@ -55,7 +55,6 @@ input[type="checkbox"],
 input[type="radio"]  {
   display: inline-block;
   margin-left: 10px;
```
Comment (Mitso) [inline tuneme/styles/tuneme/_base.scss]: @moh-moola please remove 
margin-left: 10px;

Comment (Mitso) [reply tuneme/styles/tuneme/_base.scss]: On the below selector:
.competition input[type="checkbox"], 
.yourwords input[type="checkbox"]
 Also remove margin-right: 10px

Comment (Mitso) [reply tuneme/styles/tuneme/_base.scss]: Please change this to display: initial;
input[type="checkbox"] + label,
input[type="radio"] + label  {
   display: inline-block;
}

And check no side effect across forms.

Comment (moh-moola) [reply tuneme/styles/tuneme/_base.scss]: .competition input[type="checkbox"],
.yourwords input[type="checkbox"] selectors are not applicable/present in tuneme

**Comment by Mitso** [comment]
🍰 🍰 🍰  👍

**Pull request merged**